en-us.yml

```diff
@@ -1,7 +1,50 @@
 # Loritta's English Translation
 # Authors:
 # MrPowerGamerBR (https://mrpowergamerbr.com/)
+#ItsGabii
+discord:
+  permissions:
+    kick-members: "Kick members"
+    ban-members: "Ban members"
+    administrator: "Administrador"
+    manage-channel: "Manage channel"
+    manage-server: "Manage server"
+    add-reactions: "Add reacions"
+    message-history: "See message history"
+    view-audit-logs: "See Audit Logs"
+    message-read: "Read messages"
+    message-write: "Send messages"
+    message-manage: "Manage messages"
+    message-ext-emoji: "Use external emojis"
+    message-embed-links: "insert links"
+    manage-roles: "Manage roles"
+    manage-permissions: "Manage permissions"
+    manage-webhooks: "Manage Webhooks"
+    manage-emotes: "Manage emojis"
+    create-instant-invite: "Create an instant invite"
 commands:
+please-wait-cooldown: "Calm down! you need to wait more **{0}** before execute another command! {1}"
+  error-while-executing-command: "Something gone extremaly wrong in executing this command... 😭 Sorry for the inconvenience. 😢"
+  cant-use-in-private: "You can't use this command in private messages!"
+  user-does-not-exists: "This user doesn't exist!"
+  doesnt-have-permission-discord: "Você não tem permissão para usar este comando! Para utilizá-lo, você precisa ter permissão para {0}!"
+  lori-doesnt-have-permission-discord: "I wish I could execute this command, but unfortunately i don't have permission to {0}... {1} Ask for an administrator give this permission for me! Thanks! {2}"
+  command-only-for-owner: "Only special people can use this command :3"
+  arguments:
+    text: "text"
+    number: "number"
+    user: "user"
+    image: "image"
+  language:
+    description: "You know that I can speak in another languages? {0} Change the language that I speak in your server!"
+    please-select-your-language: "Choose the wished language!"
+    translated-by: "Translated by"
+    language-changed: "Ready, now I'am going to speak {0} in this server!"
+  ajuda:
+    error-while-opening-dm: "I wanted to send the Help to you,but how I am supossed to send if you disabled your private messages between people of servers that you know. http://i.imgur.com/Z1HwW2j.png 🤷"
+  roll:
+    description: "Rolls a dice and tells the result, perfect to when you want to play Monopoly but you lost your dice."
+    how-much-sides: "How many sides the dice I will roll have"

```

Comment:
How many sides the dice will have

Reply:
`How many sides should the dice have`
acho q fica melhor